cinema/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=60)
+    description = models.TextField(null=True)
+    duration = models.IntegerField()
```

Comment:
it's a good practice to add str method to models

Reply:
yes, I have forgotten to add that